tests/test_playbooks/fixtures/content_view_version-0.yml

```diff
@@ -4,22 +4,16 @@ interactions:
     headers:
       Accept:
       - application/json;version=2
-      Accept-Encoding:
-      - gzip, deflate
-      Connection:
-      - keep-alive
-      User-Agent:
-      - apypie (https://github.com/Apipie/apypie)
     method: GET
     uri: https://foreman.example.com/api/status
   response:
     body:
-      string: !!python/unicode '{"result":"ok","status":200,"version":"1.22.0","api_version":2}'
+      string: !!python/unicode '{"result":"ok","status":200,"version":"1.21.3","api_version":2}'
     headers:
+      apipie-checksum:
+      - d841c30f0b33dc0af1a12bd583b7add23cc87b95
```

Comment:
so it seems `filter_header` didn't work in https://github.com/theforeman/foreman-ansible-modules/blob/master/tests/vcr_python_wrapper.py#L125 :(

@mdellweg help appreciated :)

Reply:
right, because that only applies to the request, not the response m)